run/deployment-previews/cloudbuild-configurations/README.md

```diff
@@ -0,0 +1,5 @@
+These configuations aren't used by this repo itself, but are configurations required to use the code in this repo. 
+
+ * cloudbuild.yaml - for main branch pushes
```

Comment:
nit: link to files and make description more descriptive :) ie Deploy service when pushed to main branch